com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalUnlitSubTarget.cs

```diff
@@ -200,7 +200,8 @@ static class UnlitKeywords
         {
             public static KeywordCollection Unlit = new KeywordCollection
             {
-                { CoreKeywordDescriptors.Lightmap },
+                { CoreKeywordDescriptors.StaticLightmap },
+                // TODO ENLIGHTEN: Should this contain lightmaps at all?
```

Comment:
problem is that without proper custom lighting solution in Shadergraph people start with the unlit then add light mapping nodes to it, if we removed light-maps from the unlit target this would ruin a lot of peoples days

Reply:
I will replace the TODO with a comment explaining this.

Reply:
fixed here 8d71165f55